localstack/services/s3/s3_listener.py

```diff
@@ -1307,5 +1320,91 @@ def return_response(self, method, path, data, headers, response, request_handler
                 response.headers['Content-Encoding'] = 'gzip'
 
 
+def authenticate_presign_url(method, path, data=None, headers={}):
+    # ignored_headers conatins headers which don't get involved in signature calculations process
+    # these headers are being sent by the localstack by default.
+    ignored_headers = [
+        'Remote-Addr', 'Host', 'User-Agent', 'Accept-Encoding',
+        'Accept', 'Connection', 'Origin',
+        'X-Forwarded-For', 'x-localstack-edge', 'Authorization'
+    ]
+
+    sign_headers = []
+    url = 'http://localhost:4566{}'.format(path)
+    parsed = urlparse.urlparse(url)
+    query_params = parse_qs(parsed.query)
+    AWS_ACCESS_KEY = 'temp'
```

Comment:
Better add these static credentials to `constants.py` and import them here:
```
TEST_AWS_ACCESS_KEY_ID = 'test'
TEST_AWS_SECRET_ACCESS_KEY = 'test'
```

Reply:
Sure